fossunited/fossunited/doctype/event_media/event_media.py

```
@@ -0,0 +1,41 @@
+            CFPSubmissionSpeaker,
+        )
+
+        duration: DF.Duration | None
+        event: DF.Link | None
+        event_name: DF.Data | None
+        proposal: DF.Link | None
+        proposal_route: DF.Data | None
+        speakers: DF.Table[CFPSubmissionSpeaker]
+        title: DF.Data | None
+        video_type: DF.Literal[
+            "Talk",
```

Ruff (0.15.20)

[error] Undefined name `Talk`

(F821)